### PR TITLE
subscribe issue

### PR DESCRIPTION
**Pull request by QvodSoldier**
#### What type of this PR
/kind feature

#### What this PR does / why we need it:
Added subscription function, But it does not include sending messages to subscribers

#### Specified Reviewers:
/assign @Effet

### CODE REVIEW

File: modules/openapi/api/apis/cmdb/cmdb_issue_unsubscribe.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import (
+	"net/http"
+
+	"github.com/erda-project/erda/modules/openapi/api/apis"
+)
+
+var CMDB_ISSUE_UNSUBSCRIBE = apis.ApiSpec{
+	Path:        "/api/issues/<id>/unsubscribe",
+	BackendPath: "/api/issues/<id>/unsubscribe",
```
Comment (Effet) [inline modules/openapi/api/apis/cmdb/cmdb_issue_unsubscribe.go:24]: /api/issues/<id>/actions/unsubscribe

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/openapi/api/apis/cmdb/cmdb_issue_unsubscribe.go (hunk 1 above)
Comment (QvodSoldier) [reply modules/openapi/api/apis/cmdb/cmdb_issue_unsubscribe.go]: done

File: modules/cmdb/dao/issue_subscriber.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import "github.com/jinzhu/gorm"
+
+// IssueSubscriber Issue's subscriber
+type IssueSubscriber struct {
+	BaseModel
+	IssueID int64
+	UserID  string
+}
+
+// TableName 表名
```
Comment (Effet) [inline modules/cmdb/dao/issue_subscriber.go:25]: english required

File: .erda/migrations/cmdb/20210610-addIssueSubscriber.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE `erda_issue_subscriber` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT COMMENT '主键',
+  `created_at` datetime DEFAULT NULL COMMENT '表记录创建时间',
```
Comment (Effet) [inline .erda/migrations/cmdb/20210610-addIssueSubscriber.sql:3]: english required

File: modules/cmdb/endpoints/issue.go [hunk 4]
```diff
@@ -731,3 +731,43 @@ func (e *Endpoints) GetIssueBugSeverityPercentage(ctx context.Context, r *http.R
...
+	identityInfo, err := user.GetIdentityInfo(r)
+	if err != nil {
+		return apierrors.ErrSubscribeIssue.NotLogin().ToResp(), nil
+	}
+
+	if err := e.issue.Subscribe(id, identityInfo); err != nil {
+		return errorresp.ErrResp(err)
+	}
+
+	// data 返回 ID
```
Comment (Effet) [inline modules/cmdb/endpoints/issue.go:751]: english required

File: modules/cmdb/endpoints/issue.go [hunk 5]
```diff
@@ -731,3 +731,43 @@ func (e *Endpoints) GetIssueBugSeverityPercentage(ctx context.Context, r *http.R
...
+	identityInfo, err := user.GetIdentityInfo(r)
+	if err != nil {
+		return apierrors.ErrSubscribeIssue.NotLogin().ToResp(), nil
+	}
+
+	if err := e.issue.Unsubscribe(id, identityInfo); err != nil {
+		return errorresp.ErrResp(err)
+	}
+
+	// data 返回 ID
```
Comment (Effet) [inline modules/cmdb/endpoints/issue.go:771]: english required

**Review by Effet — CHANGES_REQUESTED** [review]

File: modules/cmdb/endpoints/issue.go (hunk 5 above)
Comment (QvodSoldier) [reply modules/cmdb/endpoints/issue.go]: done

File: modules/cmdb/endpoints/issue.go (hunk 4 above)
Comment (QvodSoldier) [reply modules/cmdb/endpoints/issue.go]: done

File: .erda/migrations/cmdb/20210610-addIssueSubscriber.sql (hunk 3 above)
Comment (QvodSoldier) [reply .erda/migrations/cmdb/20210610-addIssueSubscriber.sql]: done

File: modules/cmdb/dao/issue_subscriber.go (hunk 2 above)
Comment (QvodSoldier) [reply modules/cmdb/dao/issue_subscriber.go:25]: done

**Review by Effet — APPROVED** [review]

**Pull request merged**